examples/first-frame-network/main.cpp

```diff
@@ -73,7 +73,7 @@ int main(int argc, char *argv[]) {
         std::cout << "no frame type available!";
         return 0;
     }
-    status = camera->setFrameType(frameTypes.front());
+    status = camera->setFrameType("depth_only");
```

Comment:
I think this change was just to test things, right?

Reply:
If we keep it this way we can use the example with jetson as well, it can be reverted.

Reply:
Also, if we keep this, it won't work for remote<->USB<->target setup because in this scenario there is only "depth_ir" (https://github.com/analogdevicesinc/aditof_sdk/blob/master/sdk/src/connections/usb/linux/usb_depth_sensor_linux.cpp#L176 and https://github.com/analogdevicesinc/aditof_sdk/blob/master/sdk/src/connections/usb/windows/usb_depth_sensor_windows.cpp#L523)

Reply:
you could set the frame type with the first element from frameTypes. This way it's more generic.

Reply:
Fixed.